docs/pages/tutorial/deploy-to-mainnet/main.md

```diff
@@ -0,0 +1,187 @@
+
+* Create `.env.sepolia` containing your RPC provider, account and private key. Make sure that account is deployed and has some [ETH](https://starknet-faucet.vercel.app) in it (0.001 is more than enough).
+
+```sh
+# usage: source .env.sepolia
+export STARKNET_RPC_URL=<RPC_PROVIDER_URL>
+export DOJO_ACCOUNT_ADDRESS=<YOUR_ACCOUNT_ADDRESS>
+export DOJO_PRIVATE_KEY=<YOUR_ACCOUNT_PRIVATE_KEY>
+```
+
+
+
+### Deployment
+
+* To load the env variables and deploy to the chain you can use this script (for mainnet, just replace `sepolia` with `mainnet`):
+
+```bash
+# Stop script on error
+set -e
+
+# Load environment variables from the appropriate file
+ENV_FILE=".env.sepolia"
+
+if [ -f "$ENV_FILE" ]; then
+  echo "Loading environment variables from $ENV_FILE..."
+  export $(grep -v '^#' "$ENV_FILE" | xargs)
+else
+  echo "Environment file $ENV_FILE not found!"
+  exit 1
+fi
+
+# Define a cleanup function to clear environment variables
+cleanup_env() {
+  echo "Cleaning up environment variables..."
+  unset STARKNET_RPC_URL
+  unset DOJO_ACCOUNT_ADDRESS
+  unset DOJO_PRIVATE_KEY
+  echo "Environment variables cleared."
+}
+
+# Set the trap to execute cleanup on script exit or error
+trap cleanup_env EXIT
+
+# Build the project
+echo "Building the project..."
+sozo -P sepolia build
+
+# Deploy the project
+echo "Deploying to Sepolia..."
+sozo -P sepolia migrate
+
+# Deployment succeeded message
+echo "Deployment completed successfully."
+```
+
+* For this script to work don't forget to give it execution permissions:
+ `chmod +x <script_name>.sh`
+
+* This localises the env variables to the deployment script, so if for any reason the deployment is aborted, it cleans up the env variables.
+
+
+* sozo will output the rpc url, account address and deployed block.
+
+```sh
+ profile | chain_id | rpc_url
+---------+----------+------------------------
+ sepolia | SN_SEPOLIA | <RPC_PROVIDER_URL>
+
+
+🌍 World deployed at block <DEPLOYED_BLOCK> with txn hash: <DEPLOYMENT_TXN_HASH>
+⛩️  Migration successful with world at address <WORLD_ADDRESS>
+```
+
+Your world is deployed! 
+
+### Torii Indexer
+
+Now, if you're building a Dojo client, you will need a Torii service to index our world...
+
+* Install [slot](https://github.com/cartridge-gg/slot) or update it
+
+```sh
+slotup
+```
+
+* Authorize
+
+```sh
+slot auth login
+```
+
+* Create Torii service with this command, replacing...
+  * `SERVICE_NAME` can be the name of the game/dapp. Once you create it, you own that name.
+  * `DOJO_VERSION`: your Dojo version (ex: `v1.0.1`)
+  * `WORLD_ADDRESS`: from your Dojo config file `dojo_sepolia.toml` or from the deployment output
+  * `RPC_URL`: your RPC provider url
+  * Take a note of the endpoints after it is deployed...
+
+```sh
+slot deployments create <SERVICE_NAME> torii --version <DOJO_VERSION> --world <WORLD_ADDRESS> --rpc <RPC_URL>
```

Comment:
```suggestion
slot deployments create <PROJECT_NAME> torii --version <DOJO_VERSION> --world <WORLD_ADDRESS> --rpc <RPC_URL>
```
Project name would be more accurate since the service name can be `torii` or `katana`.